proto/well_known_types.bzl

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
```

Comment:
Actually I do have one change to request: could you create a new `wkt` subdirectory (`//proto/wkt`) and move this bzl file there, along with the call to `gen_well_known_types`? I think it makes sense for them to be in a separate package from the compilers.